isis/src/clipper/objs/ClipperNacRollingShutterCamera/ClipperNacRollingShutterCamera.truth

```diff
@@ -1,31 +0,0 @@
-Unit Test for ClipperNacRollingShutterCamera...
-Testing with test image...
-FileName:  "simulated_clipper_eis_nac_rolling_shutter.cub"
-CK Frame:  -159011
-
-Kernel IDs: 
-CK Frame ID =  -159011
-CK Reference ID =  -159010
-SPK Target ID =  -159
-SPK Reference ID =  1
-
-Focal Length =  150.401990000000012
-
-For upper left corner ...
-DeltaSample = ERROR
-DeltaLine = ERROR
-
-For upper right corner ...
-DeltaSample = ERROR
-DeltaLine = ERROR
-
-For lower left corner ...
-DeltaSample = ERROR
-DeltaLine = ERROR
-
-For lower right corner ...
-DeltaSample = ERROR
-DeltaLine = ERROR
-
-For center pixel position ...
-ERROR
```

Comment:
Thought this was odd that the truth data was expecting all of these to output ERROR.